Zend/tests/cwd_is_slash_at.phpt

```diff
@@ -40,18 +40,18 @@ if ($fp === false) {
 }
 try {
     $splFileObject = new \SplFileObject($path);
-    echo $splFileObject->getPathname(), PHP_EOL;
+    echo mb_convert_encoding($splFileObject->getPathname(), 'UTF-8', 'CP932'), PHP_EOL;
 } catch (\RuntimeException $e) {
     echo 'Failure: SplFileObject', PHP_EOL;
 }
 echo mb_convert_encoding(
     shell_exec(mb_convert_encoding($php.' "表/ン/機能.php"', 'CP932', 'UTF-8')),
     'UTF-8',
     'CP932'
-);
+), PHP_EOL;
+//shell_exec(mb_convert_encoding('rmdir /S /Q "表"', 'CP932', 'UTF-8'));
```

Comment:
Yeah, the cleanup is needed as well. Otherwise the test is not reentrant, on the second run it's likely to produce warning about already existing files, etc.

Thanks.